src/internal/connector/onedrive/collections.go

```diff
@@ -743,6 +759,11 @@ func (c *Collections) UpdateCollections(
 				continue
 			}
 
+			collectionKind := CollectionScopeFolder
```

Comment:
Nit: naming consistency
```suggestion
			colScope := CollectionScopeFolder
```